go.mod

```diff
@@ -3,165 +3,159 @@ module knative.dev/client
 go 1.17
 
 require (
-	github.com/emicklei/go-restful v2.15.0+incompatible // indirect
 	github.com/google/go-cmp v0.5.9
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hashicorp/hcl v1.0.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/spf13/afero v1.8.0 // indirect
-	github.com/spf13/cobra v1.4.0
+	github.com/spf13/afero v1.9.2 // indirect
+	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.10.1
-	golang.org/x/mod v0.5.1
-	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
-	gotest.tools/v3 v3.1.0
-	k8s.io/api v0.23.5
-	k8s.io/apiextensions-apiserver v0.23.4
-	k8s.io/apimachinery v0.23.5
-	k8s.io/cli-runtime v0.23.4
-	k8s.io/client-go v0.23.5
-	k8s.io/code-generator v0.23.5
-	knative.dev/eventing v0.32.0
-	knative.dev/hack v0.0.0-20220524153203-12d3e2a7addc
-	knative.dev/networking v0.0.0-20220524205304-22d1b933cf73
-	knative.dev/pkg v0.0.0-20220524202603-19adf798efb8
-	knative.dev/serving v0.32.0
+	github.com/spf13/viper v1.13.0
+	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
+	golang.org/x/term v0.0.0-20220919170432-7a66f970e087
+	gotest.tools/v3 v3.3.0
+	k8s.io/api v0.25.2
+	k8s.io/apiextensions-apiserver v0.25.2
+	k8s.io/apimachinery v0.25.2
+	k8s.io/cli-runtime v0.25.2
+	k8s.io/client-go v1.5.2
+	k8s.io/code-generator v0.23.9
+	knative.dev/eventing v0.35.0
+	knative.dev/func v1.8.0
```

Comment:
@lance whatever nice display version we'd like to have here, e.g. `v1.8.1` maybe. That's displayed with `kn func version`.